### PR TITLE
EDU-1067 Improve resource selector

### PR DESCRIPTION
**Pull request by adrianaluchian**
Closes https://educandu.atlassian.net/browse/EDU-1067

### CODE REVIEW

File: src/components/resource-picker/storage-location.js [hunk 1]
```diff
@@ -123,29 +79,38 @@ function StorageLocation({
...
-            elementType={Search}
-            placeholder={t('common:search')}
-            value={typedInSearchTerm}
-            onSearch={handleSearchClick}
-            onChange={handleSearchTermChange}
+          <Input
+            allowClear
+            value={filterText}
+            placeholder={t('filterPlaceholder')}
+            onChange={handleFilterTextChange}
```
Comment (ahelmberger) [inline src/components/resource-picker/storage-location.js:98]: One thing I did in my current ticket (which has a real-time table filter behaving in the exact same way) was adding `prefix={<SearchOutlined />}` which looks nice (I know it's technically no search, but people will use it to search for something). Maybe we could align on it (or something similar)?

File: src/components/resource-picker/storage-location.yml [hunk 2]
```diff
@@ -1,18 +1,18 @@
...
 filesView_list:
-  en: List
-  de: Liste
+  en: List view
+  de: Listenansicht
 filesView_grid:
-  en: Grid
-  de: Grid
+  en: Grid view
+  de: Grid-Ansicht
```
Comment (ahelmberger) [inline src/components/resource-picker/storage-location.yml:9]: We could go with "Kachelansicht" in German here

**Comment by ahelmberger** [comment]
Works very nicely :+1:

File: src/components/resource-picker/storage-location.js (hunk 1 above)
Comment (adrianaluchian) [reply src/components/resource-picker/storage-location.js:99]: Good idea, added it too 👍 
I tried using the AntD `FilterOutlined` icon but it does not look good enough to make it intuitive:
Screen Shot 2023-01-26 at 16 06 22

File: src/components/resource-picker/storage-location.yml (hunk 2 above)
Comment (adrianaluchian) [reply src/components/resource-picker/storage-location.yml]: Went with, thanks!

File: src/components/resource-picker/storage-location.js (hunk 1 above)
Comment (ahelmberger) [reply src/components/resource-picker/storage-location.js:99]: That's exactly what I tried too but realized I perceive it as a speaker box that some rotated :stuck_out_tongue_closed_eyes:

**Review by ahelmberger — APPROVED** [review]

**Pull request merged**